cainafrica/src/components/Footer.js

```diff
@@ -84,43 +84,73 @@ const Footer = () => {
             <InnerFooterBox>
                 <MailBox>
                     <h2>CONTACT US</h2>
-                    <p>cainafrica@gmail.com</p>
-                    <p>Subscribe to our 
-                        <strong><a href="http://eepurl.com/dyu7RL" target="_blank"  rel="noopener noreferrer" style={{textDecoration:"none"}} > Newsletter here</a></strong> 
-                    </p>
-                </MailBox>
-
-                <LogoBox>
-                    <Logo src={CainWhiteLogo} alt="Cain logo"/>
-                    
-                    <InlineFollowButtons       //From sharethis-reactjs Library
+                     <p style={{ marginTop: 0, marginBottom: "8px" }}>info@cainafrica.org</p>
+                     <p style={{ marginTop: 0, marginBottom: "8px" }}>+2347049879535</p>
+                     <div className="social-icons" style={{ display: 'flex', alignItems: "center"}}> 
+                    <InlineFollowButtons //From sharethis-reactjs Library
                         config={{
-                        action: 'Follow us:', // call to action (STRING)
-                        action_enable: false,  // show/hide call to action (true, false)
-                        action_pos: 'bottom', // position of call to action (left, top, right)
-                        alignment: 'center',  // alignment of buttons (left, center, right)
-                        color: 'social',       // set the color of buttons (social, white)
-                        enabled: true,        // show/hide buttons (true, false)
-                        networks: [           // which networks to include (see FOLLOW NETWORKS)
-                            'twitter',
-                            'facebook',
-                            'instagram',
-                            'linkedin',
-                            'youtube'
+                        action: "Follow us:", // call to action (STRING)
```

Comment:
Most of these lines have no change except changing single quote ' to double quote ". Please can you keep them all as single quote?